-1

Tested at: d7680b20def6b4eddcdb552ca75a1b4cfead57d7

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14502/12987/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-05-12-2300'
config/SCRAM/GMake/Makefile.rules:1935: recipe for target 'src' failed
gmake: *** [src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre>

--- a/FWCore/ParameterSet/src/ParameterDescription.cc
+++ b/FWCore/ParameterSet/src/ParameterDescription.cc
@@ -645,6 +645,7 @@ namespace edm {
         writeValueInVector<T>(os, value_[0], format);
       } else if(value_.size() >= 1U) {
         if(format == DOC) os << "(vector size = " << value_.size() << ")";
+        if(format == CFI and value_.size() > 255U) os << " *(";
         os.fill(' ');
         bool startWithComma = false;
         int i = 0;
@@ -656,6 +657,7 @@ namespace edm {
                                     format,
                                     std::ref(i)));
         if(format == CFI) os << "\n" << std::setw(indentation) << "";
+        if(format == CFI and value_.size() > 255U) os << ") ";
       }
       os.flags(ff);
       os.fill(oldFill);

